raiden/network/proxies/token_network.py

```diff
@@ -201,8 +201,8 @@ def _new_netting_channel(self, partner: typing.Address, settle_timeout: int):
 
         return transaction_hash
 
-    def channel_exists(self, partner: typing.Address) -> bool:
-        channel_data = self.detail_channel(partner)
+    def channel_exists(self, participant: typing.Address, partner: typing.Address) -> bool:
```

Comment:
please use `participant1: typing.Address, participant2: typing.Address` to be consistent through out the class, there are multiple places that need to be updated.